jobs/kpi-forecasting/kpi-forecasting/Utils/FitForecast.py

```diff
@@ -1,16 +1,18 @@
-from datetime import datetime
-import typing
```

Comment:
you should not be removing this import, see line 13

Reply:
I didn't remove it :) This PR only alphabetized the imports and removed some extra line breaks; `import typing` is now on line 5 of the changed file.